screens/AddListScreen.tsx

```diff
@@ -62,6 +67,7 @@ class AddList extends React.Component<Props, State> {
 
     if (response.ok) {
       Alert.alert("Hurray!", body.message ?? "Your List has been added.");
+      await this.props.route.params.fetchLists();
```

Comment:
```suggestion
      this.props.route.params.fetchLists();
```
By awaiting the function, the pop doesn't happen right away. So I thought I had to manually go back. Then, when the pop executes, it takes me back to login.